tests/unit/routes/test_deployment.py

```diff
@@ -41,6 +41,17 @@ def _mock_online_enable_service_update_data_warehouse(self, mock_deployment_flow
 
     @pytest.fixture(autouse=True)
     def _always_patch_app_get_storage_fixture(self, storage):
+        """
+        Patch app.get_storage for all tests in this module
+        """
+        with patch(
+            "featurebyte.routes.deployment.controller.DeploymentController._validate_deployment_is_online_enabled",
+            return_value=None,
+        ):
+            yield
```

Comment:
nit: I think this is supposed to be under `_always_patch_deployment_online_fixture`?